board/piksiv3/rootfs-overlay/etc/ntp.conf.gps

```diff
@@ -0,0 +1,7 @@
+server 127.127.20.0 mode 1 true
```

Comment:
Where does this value come from? `127.127.20.0`?

Reply:
I guess this is a standard ntpd thing, it looks like it reads from /dev/gps0 and add a "server" for gps at 127.127.20.0?  Would appreciate a comment to that effect for the uninitiated.

http://support.ntp.org/bin/view/Support/ConfiguringNMEARefclocks
http://wiki.gude.info/Ntpd_Installation